.github/workflows/push.yml

```
@@ -193,6 +193,7 @@ jobs:
         uses: ./.github/actions/setup-yarn
         with:
           RUNS_ON_S3_BUCKET_CACHE: ${{ secrets.S3_DOCKER_CACHE_BUCKET }}
+          YARN_NPM_REGISTRY_SERVER: ${{ secrets.YARN_NPM_REGISTRY_SERVER }}
```

actionlint (1.7.4)

input "YARN_NPM_REGISTRY_SERVER" is not defined in action "Setup yarn" defined at "./.github/actions/setup-yarn". available inputs are "RUNS_ON_S3_BUCKET_CACHE"

(action)